workflow/Utils/utils.go

```diff
@@ -100,19 +95,14 @@ func VerifyIfDriver(activeRole any) error {
 	return nil
 }
 
-func CreateAuthorizedRequest(url string, jsonData []byte, c *gin.Context, MethodType string) (*http.Request, error) {
+func CreateAuthorizedRequest(url string, jsonData []byte, MethodType string, token string) (*http.Request, error) {
```

Comment:
`NewRequest`.
Can combine CreateAuthorizedRequest and CreateRequest.

Reply:
i have combined the CreateAuthorizedRequest and CreateRequest.